### PR TITLE
ModuleInterface: rephrase remark message when acquiring lock file failed

### PR DESCRIPTION
**Pull request by nkcsgexi**
The original remark message “could not acquire lock file for module interface” sounds too severe.
It may confuse users that this is a serious issue. We should rephrase it to a less obtrusive one.

rdar://70055223

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

File: include/swift/AST/DiagnosticsFrontend.def [hunk 1]
```diff
@@ -393,7 +393,7 @@ ERROR(error_creating_remark_serializer,none,
       "error while creating remark serializer: '%0'", (StringRef))
 
 REMARK(interface_file_lock_failure,none,
-      "could not acquire lock file for module interface '%0'", (StringRef))
+      "building module interface without lock file", ())
```
Comment (xymus) [inline include/swift/AST/DiagnosticsFrontend.def:396]: That's much less scary, I like it!

Would it be useful to say what is the drawback here, like "this may oversubscribe the CPU"? But I'm not sure if it would be reassuring knowing why me remark on lock files or bring more confusion tbh.

**Review by xymus — APPROVED** [review]

File: include/swift/AST/DiagnosticsFrontend.def (hunk 1 above)
Comment (nkcsgexi) [reply include/swift/AST/DiagnosticsFrontend.def:396]: yeah, I don't think this additional information offers much value to the users.
